.changeset/twelve-jars-pull.md

```diff
@@ -0,0 +1,5 @@
+---
+"@latticexyz/store": minor
+---
+
+Added the type foundation for the new store config. Not all types are included yet and the runtime implementation is still missing. Those will be added in followup PRs.
```

Comment:
do we wanna leave out the changeset for now since we're not wanting folks to consume this quite yet?

Reply:
yeah could do!

Reply:
removed